### PR TITLE
23 teams

### PR DESCRIPTION
**Pull request by hypha**

### CODE REVIEW

File: data/pages-content/thankyou.md [hunk 1]
```diff
@@ -1,310 +1,341 @@
...
-- Aurélie Vancraeyenest
+- Agata Migalska
+- Alexander Hess
+- Alisa
+- Andrii Soldatenko
+- Christian Ledermann
+- Cristián Maureira-Fredes
+- Clinton Roy
 - Cosima Meyer
+- Cristián Maureira-Fredes
```
Comment (cmaureir) [inline data/pages-content/thankyou.md:110]:
```suggestion
- Clinton Roy
- Cosima Meyer
- Cristián Maureira-Fredes
```
removing duplication

File: data/pages-content/thankyou.md [hunk 2]
```diff
@@ -1,310 +1,340 @@
...
 subtitle: Thank you
 ---
 
-# Thank you
+# Thank YOU
 
-The journey to Dublin was a 2-year arduous wait. But we made it together and
-it's a wrap! Big thanks to everyone who co-created EuroPython 2022, in Dublin &
-Remote.
+The journey to Prague encompassed a full year of exploration, diligent effort, and unwavering dedication. Our exceptional team made it together with warmth, compassion, humour and creativity! A heartfelt appreciation extends to all those who was part of crafting EuroPython 2023 Prague & Remote.
```
Comment (rodrigogiraoserrao) [inline data/pages-content/thankyou.md:8]: There was a problem with the original. I _think_ this is better.
```suggestion
The journey to Prague encompassed a full year of exploration, diligent effort, and unwavering dedication. Our exceptional team made it together with warmth, compassion, humour and creativity! A heartfelt appreciation extends to all those who took part in crafting EuroPython 2023 Prague & Remote.
```

File: data/pages-content/thankyou.md [hunk 3]
```diff
@@ -1,310 +1,340 @@
...
 
-And the small and big design stuff you saw including t-shirts, badges, banners,
-stickers and social cards were scrambled together with love by our Design team:
 
-- Laís Carvalho
+## Design
+
+### Conference Designers
+
+Our logo, website assets, t-shirts, banners, badges, lanyards, and general aesthetics of EuroPython 2023 are created by our designers
```
Comment (rodrigogiraoserrao) [inline data/pages-content/thankyou.md:58]:
```suggestion
Our logo, website assets, t-shirts, banners, badges, lanyards, and general aesthetics of EuroPython 2023 were created by our designers
```

File: data/pages-content/thankyou.md [hunk 4]
```diff
@@ -1,310 +1,340 @@
...
 - Raquel Dou
 
 ## Financial Aid team
 
-If you are one of the 94 people who participated EuroPython 2022 either in
-Dublin or remotely, you have surely exchanged many emails with some of our
-Finaid team members. From spreadsheet wizardry, to responding to countless
-emails and processing all the receipts, here are our kind and organised humans
-behind the Finaid team:
+If you are one of the 107 people who participated EuroPython 2023 in Prague or remotely, you have surely exchanged many emails with some of our Financial Aid team members. From spreadsheet wizardry, to responding to countless
```
Comment (rodrigogiraoserrao) [inline data/pages-content/thankyou.md:70]:
```suggestion
If you are one of the 107 people who participated in EuroPython 2023 in Prague or remotely, you have surely exchanged many emails with some of our Financial Aid team members. From spreadsheet wizardry, to responding to countless
```

**Comment by Cheukting** [comment]
May I add the Beginner's Orientation by Martin Borus and me? and I am happy to be excluded from the Humble Data workshop as it is mostly done by Lais and Jodie

**Comment by hypha** [comment]
> May I add the Beginner's Orientation by Martin Borus and me? 

@Cheukting of course! I missed it! Added in https://github.com/EuroPython/website/pull/396/commits/5eeae4162ebe6b99fd207371bab32605c6078910 and thank you for the reminder! <3

File: data/pages-content/thankyou.md [hunk 5]
```diff
@@ -1,368 +1,378 @@
...
 
-## Communications & Design
+## Communications
 
-When you read the (hopefully clear but definitely witty) content on our website,
-blog, newsletter & social media, it was written by one of these wordsmith from
-our Communications team:
+As you peruse the (hopefully lucid but always witty) content on our website, blog, newsletter, and social media, it was created by one of the wordsmiths/meme crafters from our Communications team:
 
+- sangarshanan
```
Comment (Kislovskiy) [inline data/pages-content/thankyou.md:48]:
```suggestion
- Sangarshanan
```

File: data/pages-content/thankyou.md [hunk 6]
```diff
@@ -1,368 +1,378 @@
...
 The talk selection process was supported by our wonderful reviewers:
 
-- Alexander CS Hendorf
-- Alisa Dammer
-- Andy Piper
-- Anmol Krishan Sachdeva
-- Aurélie Vancraeyenest
+- Agata Migalska
+- Alexander Hess
+- Alisa
```
Comment (Kislovskiy) [inline data/pages-content/thankyou.md:104]: Just noticed that Alisa is missing the family name. Probably it should be `Alisa Dammer`

File: data/pages-content/thankyou.md [hunk 7]
```diff
@@ -1,368 +1,378 @@
...
-- Sayan Chowdhury
+The registration discord bot was developed and deployed by:
+- Artem Kislovskiy
+- Sebastiaan Zeeff
+- Cheuk Ting Ho
+- Cristián Maureira-Fredes
+- Kasisnu Singh
+- Dhanshree Arora
+- Martin Borus
+<!-- - FIXME - Ask Artem if I am missing someone. -->
```
Comment (Kislovskiy) [inline data/pages-content/thankyou.md:168]: All good, @artcz do you want to be in the list 😀?

File: data/pages-content/thankyou.md [hunk 8]
```diff
@@ -1,368 +1,378 @@
...
-- Justine Moulin
-- Mick Twomey-Lee
-- Nicholas H. Tollervey
-- Stefanie Cruz
+### Social Events Organisers
 
-### [Social Event Organisers](https://ep2022.europython.eu/humble-data)
+#### [Thursday Social Event](/social-event)
+- Artem Kislovskiy
+- sangarshanan
```
Comment (Kislovskiy) [inline data/pages-content/thankyou.md:249]:
```suggestion
- Sangarshanan
```

File: data/pages-content/thankyou.md [hunk 9]
```diff
@@ -1,368 +1,378 @@
...
-The conference wouldn't be possible without:
+We would like to thank the developers and companies behind the tools we use. The conference wouldn't be possible without:
 
-- [Pretix](https://pretix.eu/about/en/) & [venueless](https://venueless.org/en/)
-  – special thanks to Dennis Lichtenthäler & Raphael Michel
+- [Pretix](https://pretix.eu/about/en/)
+  – special thanks to Raphael Michel & Martin Gross
 - [Pretalx](https://pretalx.com/p/about/) – special thanks to Tobias Kunze
 - [1Password](https://1password.com/)
 - [Zammad](https://zammad.com/en) (self-hosted)
```
Comment (Kislovskiy) [inline data/pages-content/thankyou.md:378]: I think, (self-hosted) can be omitted

Comment (hypha) [reply data/pages-content/thankyou.md]: Thank you for the review @Kislovskiy! Addressed in https://github.com/EuroPython/website/pull/396/commits/4a5272cda88485783a4c716feb278b4c65ed296b :blush:

File: data/pages-content/thankyou.md (hunk 6 above)
Comment (hypha) [reply data/pages-content/thankyou.md]: added in https://github.com/EuroPython/website/pull/396/commits/4a5272cda88485783a4c716feb278b4c65ed296b :)

File: data/pages-content/thankyou.md [hunk 10]
```diff
@@ -1,370 +1,375 @@
...
 
+- Sangarshanan
 - Vaibhav Srivastav (VB)
 - Diego Russo
-- Sangarshanan
-- Artur Czepiel
-- Laís Carvalho
-- Vicky Twomey-Lee
-- Yasmin Morshed
+- Nicholas Tollervey
```
Comment (Vaibhavs10) [inline data/pages-content/thankyou.md:52]: Add Raquel Dou here.

File: data/pages-content/thankyou.md [hunk 11]
```diff
@@ -1,370 +1,375 @@
...
-These humans quietly deploy and manage our services, and are perhaps often
-overlooked until things break:
-
-- Artur Czepiel
-- Silvia Uberti
-- Patrick Arminio
-- Francesco Pierfederici
+- Cheuk Ting Ho
+- Mannie Young
+- Manoj Pandey
```
Comment (Vaibhavs10) [inline data/pages-content/thankyou.md:78]: Add Raquel Dou here.

Comment (hypha) [reply data/pages-content/thankyou.md:78]: A bit confused. Where should I add Raquel to?

File: data/pages-content/thankyou.md (hunk 7 above)
Comment (hypha) [reply data/pages-content/thankyou.md]: I forgot to add Artur. Fixed now here: https://github.com/EuroPython/website/pull/396/commits/d1674762380730a38ef6bed15a3debc8c42aab57

**Pull request merged**